python/vyos/utils/process.py

```diff
@@ -60,6 +75,14 @@ def popen(command, flag='', shell=None, input=None, timeout=None, env=None,
     if not debug.enabled(flag):
         flag = 'command'
 
+    # Must be run as root to execute command in VRF or network namespace
+    if vrf or netns:
+        if os.getuid() != 0:
+            raise OSError('Permission denied')
```

Comment:
```suggestion
            raise OSError('Permission denied: cannot execute commands in VRF and netns contexts as an unprivileged user')
```

Maybe make the error more specific?

Reply:
Fixed